index.html

```diff
@@ -10,21 +10,19 @@
 	    <h1>Rock Collection</h1>
 	    <div id="myRocks">
 	    <table>
-	    	<thead>
-	    	<div class="table-header">
+	    	<thead  class="table-header">
 		    	<tr>
 			        <th> Name </th>
 			        <th> Color </th>
 			        <th> Transparency </th>
 			        <th> Luster </th>
 			        <th> Description </th>
 		        </tr>
-	         </div>
 	       </thead>
 	     <tbody>
 	     	<tr
-	     	v-for="(rock,index) in rocks"
-	     	v-bind:class="{active:isActive, rockCollection.rank>6}">
+	     	v-for="rock in rocks"
+	     	v-bind:class="{active:isActive, favorites: rockCollection.rank>6}">
```

Comment:
you need a key to precede the condition so I added favorites. That said, it isn't really applying this class.